runtime/interpreter/decode.go

```diff
@@ -1571,10 +1609,6 @@ func (d *DecoderV5) decodeCapabilityStaticType() (StaticType, error) {
 // This also extracts out the fields raw content and cache it separately inside the value.
 //
 func decodeCompositeMetaInfo(v *CompositeValue, content []byte) error {
-	if v.encodingVersion < 5 {
-		return decodeCompositeMetaInfoV4(v, content)
-	}
```

Comment:
I removed the delegation here and in similar places, as we don't need to be backwards-compatible in the new encoder and decoder, because we are going to migrate all of the state